pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -117,8 +118,15 @@ def work(self):
                         format_time((pokestop_cooldown / 1000) -
                                     seconds_since_epoch)))
             elif spin_result == 4:
-                logger.log("Inventory is full, switching to catch mode...", 'red')
-                self.config.mode = 'poke'
+                logger.log("Inventory is full, drop items...", 'red')
+                #self.config.mode = 'poke'
+                self.api.get_player()
+                response_dict_player = self.api.call()
+                player = response_dict_player['responses']['GET_PLAYER']['player_data']
+                items_count = self.bot.get_inventory_count('item')
+                items_stock = self.bot.current_inventory()
+                worker = DropItemsWorker(self.bot, items_count, player['max_item_storage'], items_stock.copy())
```

Comment:
We shouldn't be calling workers from workers. Please put this in the __init__ if it isn't already and have it run on every tick.